pkg/reconciler/testing/v1/parallel.go

```diff
@@ -113,3 +116,38 @@ func WithFlowsParallelAddressableNotReady(reason, message string) FlowsParallelO
 		p.Status.MarkAddressableNotReady(reason, message)
 	}
 }
+
+func WithFlowsParallelEventPoliciesReady() FlowsParallelOption {
+	return func(p *flowsv1.Parallel) {
+		p.Status.MarkEventPoliciesTrue()
+	}
+}
+
+func WithFlowsParallelEventPoliciesNotReady(reason, message string) FlowsParallelOption {
+	return func(p *flowsv1.Parallel) {
+		p.Status.MarkEventPoliciesFailed(reason, message)
+	}
+}
+
+func WithFlowsParallelEventPoliciesListed(policyNames ...string) FlowsParallelOption {
+	return func(p *flowsv1.Parallel) {
+		for _, name := range policyNames {
+			p.Status.Policies = append(p.Status.Policies, eventingduckv1.AppliedEventPolicyRef{
+				APIVersion: v1alpha1.SchemeGroupVersion.String(),
+				Name:       name,
+			})
+		}
+	}
+}
```

Comment:
Can we move this function to the bottom? As I think it is better to group similar functions together. (i.e `WithFlowsParallelEventPoliciesReady`, `WithFlowsParallelEventPoliciesNotReady`, `WithFlowsParallelEventPoliciesReadyBecauseOIDCDisabled` and `WithFlowsParallelEventPoliciesReadyBecauseNoPolicyAndOIDCEnabled` are all about marking the EventPolicies Status)

Reply:
sure, done PTAL!